examples/apps/cloudfront-http-redirect/testEvent.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Seems meaningless to include an event for this app. You can simply run [sam local invoke --no-event](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-local-invoke.html) without an event.

Applies to all apps that empty event files were added.

Reply:
At first it was hard to tell which examples could run with `--no-event` so I added the empty event to make it explicit that the lambda function does not interact with the incoming event in any way.

Maybe the intent of that empty test event is not as intuitive as I thought. I can remove it

Reply:
I agree those empty events are weird, I've removed them